Laravel/root/routes/web.php

```diff
@@ -13,6 +18,54 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
+//管理画面トップページ
+Route::get('/admin', function () {
+    return view('/admin/top');
+});
+//admin側社員管理
+Route::get('/users', [UserController::class, 'index'])->name('users');
+Route::get('user/register', [UserController::class, 'create'])->name('user.register.create');
+Route::post('user/register', [UserController::class, 'store'])->name('user.register.store');
+Route::get('user/{user}', [UserController::class, 'show'])->name('user.show');
+Route::get('user/{user}/edit', [UserController::class, 'edit'])->name('user.edit');
+Route::patch('user/{user}', [UserController::class, 'update'])->name('user.update');
+Route::post('user/{user}/confirm', [UserController::class, 'confirm'])->name('user.confirm');
+Route::delete('user/{user}', [UserController::class, 'destroy'])->name('user.destroy');
+//admin側勤怠管理
+Route::prefix('admin/works')->group(function () {
+    Route::get('/',[AdminWorkController::class,'index'])->name('admin.works');
+    Route::post('/',[AdminWorkController::class,'search'])->name('admin.works.search');
+    Route::get('/show/{user}/{searchYear}/{searchMonth}/{name}',[AdminWorkController::class,'show'])->name('admin.works.show');
+    Route::get('/show/{work}/{user}/{searchYear}/{searchMonth}/{name}',[AdminWorkController::class,'approval'])->name('admin.works.approval');
+    Route::get('/csv/{id}/{searchYear}/{searchMonth}/{name}',[AdminWorkController::class,'csv'])->name('admin.works.csv');
+});
+
+//ログイン画面(社員)
+Route::middleware(['guest'])->group(function () {
+
+    Route::get('/login', [AuthController::class, 'showLogin'])->name('showLogin');
+    Route::post('/login', [AuthController::class, 'login'])->name('login');
+});
+
+//ログイン後画面（社員）
+Route::middleware(['auth'])->group(function () {
+    //勤怠管理トップページ
+    Route::get('/home', function () {
+        return view('/user/top');
+    })->name('home');
+    //勤怠管理
+    Route::prefix('work')->group(function () {
+        Route::get('/{user}', [WorkController::class, 'index'])->name('work'); //一覧表示
```

Comment:
[NITS]
nameの末尾には、Controllerのメソッド名にしておくと
わかりやすくなります。


```suggestion
        Route::get('/{user}', [WorkController::class, 'index'])->name('work.index'); //一覧表示
```

Reply:
nameの末尾をControllerのメソッド名になるように修正しました。